src/components/Doc/styles.module.scss

```diff
@@ -1,5 +1,18 @@
 main.DocWrapper {
   padding: 0 2rem 4rem;
+  color: #bbb;
+
+  blockquote {
+    margin: 0;
+    padding: 24px;
+    color: #bbb;
```

Comment:
Optional: Seems like we're using css variables for the rest of the colours, might be best to match here